src/hotspot/share/gc/shenandoah/shenandoahTaskqueue.inline.hpp

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (c) 2016, 2019, Red Hat, Inc. All rights reserved.
```

Comment:
I don't think we're meant to update other companies' copyrights?

Reply:
That is correct - unless requested by someone representing that copyright owner.

Reply:
Fixed.